### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Clauvet1**
This is my project

### CODE REVIEW

File: footer.php [hunk 1]
```diff
@@ -0,0 +1 @@
+<footer>Copy Rights &copy; 2022 by Clauvet</footer>
```
Comment (CollinsTatang) [inline footer.php]: - [x] Please include your footers to the different pages as you did for the header side-navbar.
### **Using the below code**
```<?php include('footer.php') ?>```

File: header.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            <img src="CSS/Images/logo.png" alt="" class="logo">
+            <h2>ADMIN</h2>
+            <nav>
+              <input type="text" placeholder="Search" title="Type in a category" id="search">
+              <ul>
+                  <li><a href="index.php" target="_blank">Dashboard</a><img src="CSS/Images/arrow.png" class="nav-arrow"></li>
+                  <li><a href="manageTickets.php" target="_blank">Manage Ticket</a><img src="CSS/Images/arrow.png" class="nav-arrow"></li>
+                  <li><a href="normalTickets.php" target="_blank">Normal Ticket</a><img src="CSS/Images/arrow.png" class="nav-arrow"></li>
+                  <li><a href="foreignerTickets.php">Foreigner Ticket</a><img src="CSS/Images/arrow.png" class="nav-arrow"></li>
+                  <li><a href="Report.php">Reports</a><img src="CSS/Images/arrow.png" class="nav-arrow"></li>
```
Comment (CollinsTatang) [inline header.php]: - [x] Please remove all the target attributes from the anchor element so that your nav links will not open a new tag on your browser.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Clauvet1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- Git/GitHub flow best practices have been followed:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.

- [ ] Please work on the responsive nature of your project to fit the different screen sizes (Moblie and Desktop) Use this article [Responsive Design](https://web.dev/learn/design/media-queries/) as a guide to make your project responsive to different screen sizes.
- [ ] Kindly add a login and registration form to your project.

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not Merge or open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: footer.php (hunk 1 above)
Comment (Clauvet1) [reply footer.php]: Ok ge

**Pull request merged**